lib/assets.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var gulpif = require('gulp-if');
-var imagemin = require('gulp-imagemin');
+import {merge} from 'lodash';
+import gulpif from 'gulp-if';
+import imagemin from 'gulp-imagemin';
 
-var defaults = {
+const defaults = {
```

Comment:
Thanks for the feedback @lyzadanger and @erikjung. It seems that `const` doesn't make sense as those default values _can_ change. I agree that it is not in the spirit of the `const` keyword. I'll change it to a `var` instead. ;)